app/src/main/java/com/architectcoders/arquitectomarvel/ui/main/pagination/CharacterAdapter.kt

```diff
@@ -31,7 +31,7 @@ class CharacterAdapter(private val listener: (Result, View) -> Unit) :
         getItem(position)?.let { characterEntity ->
             holder.bind(characterEntity.toDomainCharacter)
             holder.itemView.setOnClickListener {
-                listener(characterEntity.toDomainCharacter, it)
+                listener(characterEntity.toDomainCharacter, holder.imageViewHero)
```

Comment:
Thanks @joseantoniocampillo could you change the hero word by character?

Reply:
Yeah, that should be changed too. Initially, we had some hero/heroes name, but that was changed to character in the project